__tests__/app.test.js

```diff
@@ -84,3 +84,30 @@ describe("GET", () => {
       });
   });
 });
+
+describe("GET", () => {
```

Comment:
Just reference that you're doing a get request to users in your tests please. just add /users/ somewhere here.